### PR TITLE
Another pit-stop checkin.

### PR DESCRIPTION
**Pull request by crobby**
Create, delete, scale are all functional.  There are still some rough edges with
respect to logging and notifications and the form validation works, but doesn't display
a visable message.  All in good time.  Also, the modal handling could use some love.

### CODE REVIEW

File: app/scripts/oshinko.js [hunk 1]
```diff
@@ -1,11 +1,121 @@
...
+      newCluster: newCluster,
+      scaleCluster: scaleCluster,
+    };
+  })
+  .factory('clusterData', [
+    '$http',
+    '$q',
+    "OSHINKO_CFG",
+    function ($http, $q, OSHINKO_CFG) {
+      console.log(OSHINKO_CFG.restPort);
```
Comment (sub-mod) [inline app/scripts/oshinko.js:76]: how to get rest url value in console ?
the config.json is only for dev setup, doesnt exists in console.

File: app/scripts/oshinkoapp.js [hunk 2]
```diff
@@ -9,4 +9,5 @@ angular.module('openshiftConsole')
         controller: 'ClustersCtrl'
       });
     }
-  ]);
\ No newline at end of file
+  ])
+  .constant("OSHINKO_CFG", _.get(window.OSHINKO_CONFIG, "oshinko", {}));
```
Comment (sub-mod) [inline app/scripts/oshinkoapp.js:13]: again discovery of rest url is an issue here.

File: app/scripts/oshinko.js [hunk 3]
```diff
@@ -228,4 +339,125 @@ angular.module('openshiftConsole')
...
+  .controller('ClusterDeleteCtrl', [
+    '$q',
+    '$scope',
+    "clusterData",
+    "$modalInstance",
+    "dialogData",
+
+    function ($q, $scope, clusterData, $modalInstance, dialogData) {
+
+      $scope.clusterName = dialogData.clusterName || "";
```
Comment (sub-mod) [inline app/scripts/oshinko.js:352]: All controllers i think should have the Auth check

File: app/scripts/oshinko.js (hunk 1 above)
Comment (crobby) [reply app/scripts/oshinko.js:76]: Yes.  I had an idea on that last night.  We can probably find the route based on lookups of route objects and find the one for oshinko-rest service.

File: app/scripts/oshinkoapp.js (hunk 2 above)
Comment (crobby) [reply app/scripts/oshinkoapp.js:13]: Yep, this is just a "pit stop" checkin.  Certainly, this will come.

File: app/scripts/oshinko.js (hunk 3 above)
Comment (crobby) [reply app/scripts/oshinko.js:352]: Indeed.  I'll add those soon.

**Pull request merged**